### PR TITLE
Add adoption task to nova role

### PR DESCRIPTION
**Pull request by mrkisaolamb**

### CODE REVIEW

File: roles/edpm_nova/tasks/adoption.yml
Comment (gibizer) [inline roles/edpm_nova/tasks/adoption.yml:8]: can we use ansible.builtin.systemd instead?

File: roles/edpm_nova/defaults/main.yml
Comment (gibizer) [inline roles/edpm_nova/defaults/main.yml:35]: would use a name that signals that these are old tripleo services. What about `edpm_old_tripleo_compute_sevices`

File: roles/edpm_nova/defaults/main.yml
Comment (gibizer) [inline roles/edpm_nova/defaults/main.yml:34]: probably copy paste

File: roles/edpm_nova/tasks/adoption.yml
Comment (gibizer) [inline roles/edpm_nova/tasks/adoption.yml:3]: hm, this variable seems to be intended to the network_config roles. Do we even foresee some sensitive logs here?

**Review by gibizer — COMMENTED** [review]
Do we want to disable these adoption tasks in the greenfield deployment? If so, we need to somehow disable the adoption tag in greenfield.

**Comment by gibizer** [comment]
I'm wondering how easy would be to create a meaningful molecule test for this task. @SeanMooney do you have some ideas how to simulate the existence of an old tripleo service in molecule?

**Comment by gibizer** [comment]
@mrkisaolamb I'm not sure depends-on works between edpm-ansible and data-plane-adoption repo, but it worth a try to see if we can run this in the adoption CI even before merge

File: roles/edpm_nova/defaults/main.yml
Comment (mrkisaolamb) [reply roles/edpm_nova/defaults/main.yml]: Done

File: roles/edpm_nova/tasks/adoption.yml
Comment (mrkisaolamb) [reply roles/edpm_nova/tasks/adoption.yml]: done, removed

File: roles/edpm_nova/tasks/adoption.yml
Comment (mrkisaolamb) [reply roles/edpm_nova/tasks/adoption.yml]: done

File: roles/edpm_nova/tasks/adoption.yml
Comment (gibizer) [inline roles/edpm_nova/tasks/adoption.yml:10]: we mask them as well with `masked: yes`

**Review by gibizer — COMMENTED** [review]
Thanks. I think we need some pidfile deletion as well. Or have you checked, stopping the service removes the pidfiles too?

File: roles/edpm_nova/tasks/adoption.yml
Comment (mrkisaolamb) [reply roles/edpm_nova/tasks/adoption.yml]: done

**Comment by mrkisaolamb** [comment]
recheck

File: roles/edpm_nova/tasks/adoption.yml
Comment (bogdando) [reply roles/edpm_nova/tasks/adoption.yml:23]: have you pushed the updated revision yet?

Comment (mrkisaolamb) [reply roles/edpm_nova/tasks/adoption.yml:23]: I re-add this shell to simplified checking if service exist from list, task with systemd was failing if service wasn't running

**Comment by mrkisaolamb** [comment]
recheck

File: roles/edpm_nova/tasks/adoption.yml
Comment (gibizer) [inline roles/edpm_nova/tasks/adoption.yml:14]: Does podman return a nonzero exitcode if it succeeds? Do we need to ignore all the exit codes?

**Comment by gibizer** [comment]
> Thanks. I think we need some pidfile deletion as well. Or have you checked, stopping the service removes the pidfiles too?

Did we settled this?

**Review by gibizer — APPROVED** [review]
So Kamil checked and the pidfiles are removed by the systemd service files. 
Also the systemd module is not used as based on Kamil it cannot stop already disabled services. 
The rest of my comments can be fixed in a follow up. So I approve this to see if it helps the adoption testing where we see virtlogd issues.

**Pull request merged**
